src/Uno.UI/UI/Xaml/Controls/PasswordBox/PasswordBox.cs

```diff
@@ -15,13 +15,14 @@ public partial class PasswordBox : TextBox
 		public const string RevealButtonPartName = "RevealButton";
 		private Button _revealButton;
 		private readonly SerialDisposable _revealButtonSubscription = new SerialDisposable();
+		private bool UseIsPasswordEnabledProperty => ReadLocalValue(IsPasswordRevealButtonEnabledProperty) != DependencyProperty.UnsetValue && ReadLocalValue(PasswordRevealModeProperty) == DependencyProperty.UnsetValue;
```

Comment:
there's also the `IsDependencyPropertySet()` extension method which achieves this